File: app2/src/main/java/com/ssz/framejava/data/local/bean/Student.java
package com.ssz.framejava.data.local.bean;

import org.greenrobot.greendao.annotation.Entity;
import org.greenrobot.greendao.annotation.Id;
import org.greenrobot.greendao.annotation.Generated;

/**
 * @author : zsp
 * time : 2019 11 2019/11/5 16:02
 */
@Entity
public class Student {
    @Id(autoincrement = false)
    private Long id;
    private String name;
    private int age;
    @Generated(hash = 352757281)
    public Student(Long id, String name, int age) {
        this.id = id;
        this.name = name;
        this.age = age;
    }
    @Generated(hash = 1556870573)
    public Student() {
    }
    public Long getId() {
        return this.id;
    }
    public void setId(Long id) {
        this.id = id;
    }
    public String getName() {
        return this.name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public int getAge() {
        return this.age;
    }
    public void setAge(int age) {
        this.age = age;
    }
}

File: testarea/src/main/java/com/ssz/testarea/kt/object/dataclass/Student.kt
package com.ssz.testarea.kt.`object`.dataclass

/**
 * @author : zsp
 * time : 2019 08 2019/8/23 16:59
 * data class 至少一个参数，且构造方法里的才会加到toString里,val 不可以修改，var 可以修改
 */
data class Student(var name : String,var age : Int,var gender :String)